src/components/Logo.tsx

```diff
@@ -0,0 +1,9 @@
+import type { Component, JSX } from "solid-js";
```

Comment:
Probably should begin reorganizing the components folder- it's getting a bit large without clear organization.